choreolib/src/main/java/choreo/ext/TriggerExt.java

```diff
@@ -0,0 +1,51 @@
+package choreo.ext;
+
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.VarHandle;
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj.event.EventLoop;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+
+public class TriggerExt extends Trigger {
+    private static final VarHandle loopHandle;
+
+    static {
```

Comment:
This reflection will be unnecessary if https://github.com/wpilibsuite/allwpilib/blob/0352a60f38c249aa66f51252d9d658a11b97de63/wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/button/Trigger.java#L28 becomes public final, or accessible through a getter.